src/svgSprite.ts

```diff
@@ -554,5 +554,22 @@ export const svgSprite = (): string => `
+  </symbol>
+
+  <symbol viewBox="0 0 24 24" id="svg-clear-calendar">
+    <path fill-rule="evenodd" clip-rule="evenodd" d="M13.5618 18.8088V20.8088H19.1454C19.6954 20.8088 20.1664 20.6132 20.5584 20.2218C20.9497 19.8298 21.1454 19.3588 21.1454 18.8088V4.80884C21.1454 4.25884 20.9497 3.78817 20.5584 3.39684C20.1664 3.00484 19.6954 2.80884 19.1454 2.80884H18.1454V1.80884C18.1454 1.25655 17.6976 0.808838 17.1454 0.808838C16.5931 0.808838 16.1454 1.25655 16.1454 1.80884V2.80884H8.14535V1.80884C8.14535 1.25655 7.69764 0.808838 7.14535 0.808838C6.59307 0.808838 6.14536 1.25655 6.14536 1.80884V2.80884H5.14536C4.59536 2.80884 4.12436 3.00484 3.73236 3.39684C3.34102 3.78817 3.14536 4.25884 3.14536 4.80884V12.8088H5.14536V8.80884H19.1454V18.8088H13.5618ZM19.1454 6.80884H5.14536V4.80884H19.1454V6.80884Z" />
+    <path d="M3.14737 14.95C2.75685 15.3405 2.75687 15.9737 3.1474 16.3642L5.70782 18.9245L3.14838 21.4839C2.75785 21.8744 2.75783 22.5076 3.14834 22.8981C3.53886 23.2886 4.17202 23.2886 4.56256 22.8981L7.12207 20.3387L9.68158 22.8981C10.0721 23.2886 10.7053 23.2886 11.0958 22.8981C11.4863 22.5076 11.4863 21.8744 11.0958 21.4839L8.53632 18.9245L11.0967 16.3642C11.4873 15.9737 11.4873 15.3405 11.0968 14.95C10.7063 14.5595 10.0731 14.5594 9.68256 14.95L7.12207 17.5103L4.56158 14.95C4.17105 14.5594 3.53788 14.5595 3.14737 14.95Z" />
+  </symbol>
+
+  <symbol viewBox="0 0 24 24" id="svg-export2">
```

Comment:
maybe it's better to rename?

Reply:
both icons for export what name better to use